crawler/src/main/java/nl/fhict/happynews/crawler/crawler/QuoteCrawler.java

```diff
@@ -70,6 +70,7 @@ void crawl() {
         for (Quote quote : entity.getContents().getQuotes()) {
             Post toAdd = new Post();
             toAdd.setSource("Inspirational Quote");
+            toAdd.setSourceName("quote");
```

Comment:
Shouldn't source name and source be reversed? Might be we do this consistently wrong though.